### PR TITLE
change logic of integration

### PR DESCRIPTION
**Pull request by superxi911**
issue [#60](https://github.com/caicloud/cyclone/issues/60)

### CODE REVIEW

**Comment by superxi911** [comment]
@gaocegege @xiaomuqiao @zoumo PTAL
I'll modify docs next step.

**Comment by superxi911** [comment]
I'll modify docs in other PR to not block this issue [#21](https://github.com/caicloud/cyclone/issues/21)

File: worker/ci/runner/docker.go [hunk 1]
```diff
@@ -78,6 +81,21 @@ func toServiceContainerConfig(dn *parser.DockerNode, b *Build) (*docker_client.C
 	var name = getNameInNetwork(dn, b)
 	var alias = dn.Name
 
+	// If the image of service config to be "BUILT_IMAGE",
+	// Cyclone will use the image built dunrning the "build" step to run a service container.
```
Comment (xiaomuqiao) [inline worker/ci/runner/docker.go]: s/dunrning/during

File: api/rest/worker_node.go [hunk 2]
```diff
@@ -106,6 +107,45 @@ func createSystemWorkerNode(request *restful.Request, response *restful.Response
 	response.WriteHeaderAndEntity(http.StatusOK, createResponse)
 }
 
+//  convertVersionFromStringToInt convert the version from string to int.
+func convertVersionFromStringToInt(version string) (int, error) {
```
Comment (xiaomuqiao) [inline api/rest/worker_node.go]: Are  there letters in version possibly?

File: scripts/tools/api.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 #host = 'http://127.0.0.1:7099/'
 #host = 'http://43.254.54.38:7099/'
 #host = 'http://118.193.143.243/'
-host = 'http://139.196.115.69:7099/'
+host = 'http://192.168.99.100:7099/'
```
Comment (xiaomuqiao) [inline scripts/tools/api.py]: Is this need to modify? the same as follow

File: scripts/tools/api.py [hunk 4]
```diff
@@ -277,14 +277,14 @@ def delete_worker_node(node_id):
     #set_event('63e3a836-bdf9-4106-8c9a-a068b5e3a987', 'success', 'well done')
 	
     #create_service('superxi', 'test')
-    service_id = 'f61a8986-5ddf-4d51-8eb3-a6cde8ed6260'
+    service_id = '7e916e31-b2d9-4c59-8b5a-fc398a1f8073'
     #set_service('superxi', service_id)
     #get_service('superxi', '871e6da4-5a5d-4fd8-bb8e-166f817ff2c9')
-    get_services('superxi')
+    #get_services('superxi')
```
Comment (gaocegege) [inline scripts/tools/api.py]: is it necessary to change the script?

File: worker/ci/runner/docker.go [hunk 5]
```diff
@@ -38,6 +38,9 @@ import (
 const (
 	// BindTemplate is the template of binds.
 	BindTemplate = "%s:%s"
+
+	// Flag of running a container which use the built image as a service.
+	BuiltImage = "BUILT_IMAGE"
```
Comment (gaocegege) [inline worker/ci/runner/docker.go]: // BuiltImage is the keyword of the image built in the build step.

is more appropriate?

File: worker/ci/runner/docker.go (hunk 1 above)
Comment (superxi911) [inline worker/ci/runner/docker.go]: ok

File: api/rest/worker_node.go (hunk 2 above)
Comment (superxi911) [inline api/rest/worker_node.go]: None, at present.

File: worker/ci/runner/docker.go (hunk 5 above)
Comment (superxi911) [inline worker/ci/runner/docker.go]: Would you recommend one？

**Comment by superxi911** [comment]
The docs has modified. @gaocegege @xiaomuqiao @zoumo PTAL

**Comment by xiaomuqiao** [comment]
LGTM

**Pull request merged**